zc_events/responses.py

```diff
@@ -166,6 +166,7 @@ def activate(self, func):
         return get_wrapped(func, _wrapper_template, evaldict)
 
     def _find_match(self, resource_type, **kwargs):
```

Comment:
this function was super unintuitive using a for/else loop. I didn't want to refactor it since I don't know how it will be have. This is my workaround.